src/script.sh

```
@@ -25,30 +25,14 @@ main() {
-        options+="--bin_size $bin_size "
-    fi
-    if [ -n "$chr_names" ]; then
-        options+="--chr_names $chr_names "
-    fi
-    if [ -n "$genome" ]; then
-        options+="--genome $genome "
-    fi
-    if [ -n "$symmetry" ]; then
-        options+="--symmetry $symmetry "
-    fi
+    options+="--min_baf $min_baf "
```

Shellcheck (0.10.0)

[warning] min_baf is referenced but not assigned.

(SC2154)